lib/client.js

```diff
@@ -112,8 +113,9 @@ class AmazonMwsClient {
         requestOpts[api.upload ? 'qs' : 'form'] = signedQuery;
 
         requestOpts.headers = headers;
+        requestOpts.encoding = 'binary';
```

Comment:
This forces request to avoid to decode returned data; otherwise unrecognized character will be removed after the request